service/ztests/query-sql-agg.yaml

```diff
@@ -0,0 +1,22 @@
+# brimdata/brim#1884
+
+script: |
+  source service.sh
+  zed lake create -q test
+  zed lake use -q test@main
+  zed lake load -q data.zson
+  zed lake query -z 'from test | SELECT count(), id GROUP BY id | sort id'
```

Comment:
```suggestion
  zapi create -q test
  zapi use -q test@main
  zapi load -q data.zson
  zapi query -z 'SELECT count(), id GROUP BY id | sort id'
```

Reply:
I'd prefer to keep using `zed lake` since if I make a change I can easily copy this to lake and vice versa.

Reply:
But this needs to use `zapi` or `zed api` in order to talk to `zed lake serve`. `zed lake` just goes straight to the file system.

Reply:
Good catch!  I missed that one.